specification/columns/skufamily.md

```diff
@@ -0,0 +1,38 @@
+# SKU Family
+
+The SKU Family column represents the common name for the family or version of the [*SKU*](#glossary:SKU).
+
+It is oftentimes the case that within a [Service Subcategory](#servicesubcategory) a [Provider](#provider) offers multiple different families of versions of SKUs. The SKU Family column provides this information to group and aggregate cost and usage data across SKUs of the same family or version.
+
+The SkuFamily column adheres to the following requirements:
+
+* The SkuFamily column is RECOMMENDED be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
+* This column MUST be of type String and MAY contain null values.
+* SkuFamily values SHOULD follow a common convention used by the provider and SHOULD be consistent across SkuId and SkuPriceId records.
+* SkuFamily MUST contain null values when SkuId is null or is not associated with a family or version.
+
```

Comment:
Should we have a line that explains when it should not be null?
```suggestion
* SkuFamily SHOULD NOT be null when SkuId is not null and is associated with a logical group of SkuSize values.
```
Or do we want to go all in and always require a value based on SkuSize?
```suggestion
* SkuFamily MUST NOT be null when SkuId is not null and SkuSize is not null.
```
I'm sure it's somewhere in between, but it feels like a gap right now.